pkg/launcher/paths.go

```diff
@@ -156,3 +157,32 @@ func nonEmptyFileExists(path string) (bool, error) {
 
 	return fileInfo.Size() > 0, nil
 }
+
+// GetOriginalLauncherExecutablePath is a convenience function to determine and verify the location of
+// the originally installed launcher executable. it uses the identifier to generate the expected path and
+// verifies file presence before returning the path
+func GetOriginalLauncherExecutablePath(identifier string) (string, error) {
+	if strings.TrimSpace(identifier) == "" {
+		identifier = DefaultLauncherIdentifier
+	}
+
+	var binDirBase string
+	var launcherExeName string
+
+	switch runtime.GOOS {
+	case "windows":
+		binDirBase = fmt.Sprintf(`C:\Program Files\Kolide\Launcher-%s\bin`, identifier)
+		launcherExeName = "launcher.exe"
+	default:
+		binDirBase = fmt.Sprintf(`/usr/local/%s/bin`, identifier)
```

Comment:
Ahhh can you throw a quick comment here or in the docblock that this won't work for NixOS, just so we don't forget if we try to use it for Linux in the future?

Reply:
ahh yep! or do you think it make more sense to just add this to `pkg_utils_windows.go`? it is only in use for these windows tasks currently

Reply:
added a comment but happy to move this over if you think that's more appropriate